lib/game.rb

```diff
@@ -0,0 +1,130 @@
+require './lib/board'
+require './lib/cell'
+require './lib/ship'
+require './lib/computer_board'
+
+class Game
+  attr_reader :board,
+              :input,
+              :player_cruiser,
+              :player_sub,
+              :computer
+
+
+  def main_menu
+    puts "Welcome to BATTLESHIP"
+    puts "To start a new game, enter 'p'. To quit game, enter 'q'."
+    print ">"
+    input = gets.chomp.downcase
+    if input == "p"
+      start_game
+    elsif input == "q"
+      puts "Ok, goodbye."
+    else
+      puts "Invalid menu input."
+      main_menu
+    end
+  end
+
+  def player_place_ship(ship)
+    placement = gets.chomp.gsub(/\s+/, "").upcase.scan(/../)
+    if @player_board.place(ship, placement) == nil
+      puts "Invalid coordinates. Reminder: a #{ship.name} must be placed using #{ship.length} valid coordinates."
+      print ">"
+      player_place_ship(ship)
+    else
+      @player_board.place(ship, placement)
+    end
+  end
+
+  def place_cruiser
+    @player_cruiser = Ship.new("Cruiser", 3)
+    require "pry"; binding.pry
```

Comment:
do we want to remove this pry?

Reply:
good catch, I'll remove the pry, commit, then push to this branch